app/backend/aquifers/urls.py

```diff
@@ -21,7 +21,47 @@
     url(r'^api/v1/aquifers/$',
         never_cache(views.AquiferListAPIView.as_view()),
         name='aquifers-list'
-    ),
+        ),
+
+    url(r'^api/v1/aquifers/(?P<aquifer_id>[0-9]+)/$',
+        never_cache(views.AquiferRetrieveUpdateAPIView.as_view()),
+        name='aquifer-retrieve-update'
+        ),
+
+    url(r'^api/v1/aquifer-codes/materials/$',
+        never_cache(views.AquiferMaterialListAPIView.as_view()),
+        name='aquifer-material-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/quality-concerns/$',
+        never_cache(views.QualityConcernListAPIView.as_view()),
+        name='quality-concern-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/vulnerability/$',
+        never_cache(views.AquiferVulnerabilityListAPIView.as_view()),
+        name='aquifer-vulnerability-code-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/subtypes/$',
+        never_cache(views.AquiferSubtypeListAPIView.as_view()),
+        name='aquifer-subtype-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/productivity/$',
+        never_cache(views.AquiferProductivityListAPIView.as_view()),
+        name='aquifer-productivity-code-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/demand/$',
+        never_cache(views.AquiferDemandListAPIView.as_view()),
+        name='aquifer-demand-list'
+        ),
+
+    url(r'^api/v1/aquifer-codes/water-use/$',
+        never_cache(views.WaterUseListAPIView.as_view()),
+        name='aquifer-water-use-code-list'
+        ),
 
     url(r'^api/v1/aquifers/(?P<aquifer_id>[0-9]+)/$',
```

Comment:
This route is the same as the one on line 26. it's also causing an error  `AttributeError: module 'aquifers.views' has no attribute 'AquiferRetrieveAPIView'`. Otherwise looks good. The branch should deploy after removing this line

Reply:
Thanks Stephen